metamorph/Server.go

```diff
@@ -208,6 +208,65 @@ func (s *Server) PutTransaction(ctx context.Context, req *metamorph_api.Transact
 	}
 }
 
+func (s *Server) PutTransactions(ctx context.Context, req *metamorph_api.TransactionRequests) (*metamorph_api.TransactionStatuses, error) {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "Server:PutTransactions")
+	defer span.Finish()
+	start := gocore.CurrentNanos()
+	defer gocore.NewStat("PutTransactions").AddTime(start)
+
+	// prepare response object before filling with tx statuses
+	ret := &metamorph_api.TransactionStatuses{}
+	ret.Statuses = make([]*metamorph_api.TransactionStatus, len(req.Transactions))
+
+
+	// for each transaction if we have status in the db already set that status in the response
+	// if not we store the transaction data and set the transaction status in response array to - STORED
+	requests := make([]*store.StoreData, 0)
```

Comment:
Better would be to init with a cap:
`requests := make([]*store.StoreData, 0, len(req.Transactions))`